packages/ecom-hooks/package.json

```
@@ -0,0 +1,51 @@
+{
+    "name": "ecom-hooks",
+    "version": "2.1.0-dev",
+    "description": "A library that provides hooks for fetching data from Commerce Cloud",
+    "author": "dev@mobify.com",
+    "license": "See license in LICENSE",
+    "engines": {
+      "node": "^14.0.0",
+      "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+      },
+      "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/hooks-slas#readme",
+      "bugs": {
+        "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+      },
+    "scripts": {
+      "build": "cross-env NODE_ENV=production internal-lib-build build && tsc --emitDeclarationOnly",
+      "build:watch": "watch 'npm run build' src/",
+      "lint": "npm run lint:js",
```

```suggestion
      "lint": "npm run lint:js && tsc --noEmit",
```

To lint both javascript and typescript files